libs/site-base/src/containers/AddAccount/Storage/GDrive/RestoreList.tsx

```diff
@@ -3,24 +3,29 @@ import { getUrlParameterByName } from '../../../../utils/localState'
 import { GoogleWalletItem } from '@thecointech/types';
 import { clientUri, GetSecureApi } from '../../../../api';
 import { log } from '@thecointech/logging';
-import { Button, ButtonProps, List } from 'semantic-ui-react';
+import { ButtonProps, List } from 'semantic-ui-react';
 import { AccountMap } from '@thecointech/shared/containers/AccountMap';
 import { AccountState } from '@thecointech/account';
 import { NormalizeAddress } from '@thecointech/utilities';
 import { Wallet } from 'ethers';
 import { defineMessage, FormattedMessage } from 'react-intl';
 import { Redirect } from 'react-router-dom';
 import styles from './styles.module.less';
+import { PageHeader } from '../../../../components/PageHeader';
+import { ButtonPrimary } from '../../../../components/Buttons';
 
 type LoadingWallet = {
   wallet: Wallet;
   name: string;
   id: string;
   exists: boolean;
 }
+
+const aboveTheTitle = defineMessage({defaultMessage: "Restore Account", description: "The above the title text for the restore account page"});
+const title = defineMessage({defaultMessage: "Accounts in your Google Drive", description: "The main title for the restore account page"});
```

Comment:
This will need to be translated once site-base translation is merged in dev

Reply:
Merged in now (just testing the merge)